src/content/developers/docs/gas/index.md

```diff
@@ -21,88 +20,104 @@ Since each Ethereum transaction requires computational resources to execute, eac
 
-Transactions can either specify a gas price using the old mechanism, or specify two other parameters:
+## Tips {#tips}
 
-- Maximum Fee per Gas: The maximum gas price the transaction can be charged.
-- Maximum Priority Fee per Gas (a.k.a. Tip): The maximum priority fee the transaction signer is willing to pay the miner per gas to be included.
-  If the base fee plus this amount is less than the maximum fee per gas, this is the priority fee. Otherwise, the priority fee is the maximum
-  fee minus the base fee.
+As the base fee (which in the previous transaction fee mechanism would have gone to the miner) is burned, a priority fee (tip) is used to incentivise miners to include a transaction in the block they are mining. Without tips, it would be economically viable for miners to mine empty blocks, as they would receive the same block reward. Under normal conditions a small tip provides miners a minimal incentive to include a transaction and high tips can provide incentive for special treatment.
 
-For example, imagine a block with a base fee of 100 gwei. The pool of available transactions contains the transactions
-in the table below. Transactions A-C are type 2, so they include both a maximum fee per gas and a maximum priority fee per gas.
-Transaction D is an older transaction type (either 0, without an access list, or 1, which does have an access list), so it only specifies
-a gas price. That gas price is used for both maximum fee per gas and maximum priority fee per gas.
+## Max Fee {#maxfee}
 
-| ID  | Maximum Fee per Gas | Maximum Priority Fee per Gas | Actual Priority Fee | Actual Gas Price | Remarks                                                         |
-| --- | ------------------: | ---------------------------: | ------------------: | ---------------: | --------------------------------------------------------------- |
-| A   |             90 gwei |                      90 gwei |                 N/A |              N/A | This transaction is not going in the block                      |
-| B   |            200 gwei |                       5 gwei |              5 gwei |         105 gwei | The priority fee is the maximum priority fee                    |
-| C   |            120 gwei |                      30 gwei |             20 gwei |         120 gwei | The priority fee is the maximum (total) fee minus the base fee  |
-| D   |            200 gwei |                     200 gwei |            100 gwei |         200 gwei | Transactions that specify gas price are charged the full amount |
+To execute a transaction on the network users are able to specify a maximum limit they are willing to pay for their transaction to be executed. This optional parameter is known as the `maxFeePerGas`. In order for a transaction to be executed the max fee must exceed the sum of the base fee and the tip.
```

Comment:
Should we also touch on the concept of a refund? I think that's a nice value add of EIP 1559.